org.tlauncher.TLauncher.metainfo.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 TLauncher Inc. -->
+<component type="desktop-application">
+  <id>org.tlauncher.TLauncher</id>
+  <name>TLauncher</name>
+  <summary>The launcher for Minecraft</summary>
+  <developer_name>TLauncher Inc.</developer_name>
+
+  <description>
+    <p>Today not many good launcher for the game Minecraft, but they are
```

Comment:
There are grammar error here and there.

Reply:
Cut and paste from the site, I’d need to do a rewrite to fix all the errors, so I just let it be.